sdk/tables/Azure.Data.Tables/tests/samples/Sample1_CreateDeleteTable.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+using Azure.Data.Tables.Tests;
+
+namespace Azure.Data.Tables.Samples
+{
+    [LiveOnly]
+    public partial class TablesSamples : TablesTestEnvironment
+    {
+        [Test]
+        public void CreateTable()
```

Comment:
I think it would help add some context if we add a comment above each significant line. ex:
`// Construct a new <see cref="TableServiceClient" /> using a <see cref="TableSharedKeCredential" />.`

`// Create a new table.`

`// Get a reference to a new instance of <see cref="TableClient" /> affinized to the table we created.`